honeybee_radiance_command/options/pcomb.py

```diff
@@ -0,0 +1,163 @@
+# coding: utf-8
+
+from .optionbase import (
+    OptionCollection,
+    BoolOption,
+    NumericOption,
+    StringOption,
+    IntegerOption,
+    TupleOption,
+    FileOption
+)
+import warnings
+
+
+class PcombOptions(OptionCollection):
+    """pcomb command options.
+
+    Also see: https://floyd.lbl.gov/radiance/man_html/pcomb.1.html
+    """
+
+    __slots__ = (
+        "_h",
+        "_w",
+        "_x",
+        "_y",
+        "_f",
+        "_e",
+        "_o",
+        "_s",
+        "_c"
+        )
+
+    def __init__(self):
+        """pcomd command options."""
+
+        OptionCollection.__init__(self)
+
+        self._h = BoolOption("h", "Reduce information header - default: False")
+        self._w = BoolOption("w", "Supress warning messages - default: False")
+        self._x = IntegerOption("x", "X resolution")
+        self._y = IntegerOption("y", "Y resolution")
```

Comment:
Shouldn't we add some validation here? Am I right that they should always be a positive number?

Reply:
Added in setter

Reply:
We implemented these types to keep the setters simple. I suggest to implement the checks here.

I didn't check the args but it should be something like this.

```suggestion
        self._x = IntegerOption("x", "X resolution", min_value=0)
        self._y = IntegerOption("y", "Y resolution", min_value=0)
```